pages/k8s/install-manual.md

```diff
@@ -0,0 +1,288 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Installing CDK manually"
+  description: How to install and customise the Charmed Distribution of Kubernetes using Juju bundles.
+keywords: lxd, conjure-up, requirements,developer
+tags: [install]
+sidebar: k8smain-sidebar
+permalink: install-manual.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The recommended way to install the
+**Charmed Distribution of Kubernetes <sup>&reg;</sup>** is using the
+**conjure-up** install tool as described in the
+['Quick start' documentation][quickstart]. However, in some cases it may be useful to
+customise the install:
+
+  - Setting specific options for the master/worker nodes
+  - Adding additional components (e.g. monitoring)
+  - Configuring storage or networking  
+  - Copying an existing configuration
+  - Testing a pre-release version
+  - ...and many more
+
+In many cases, using the bundle install method outlined here will be faster and more
+repeatable than customising a deployment post-install.
+
+This documentation first presents the method for installing from the official release bundles, then explains how the bundle can be customised.
+
+
+## Install CDK from the official bundle
+
+The following sections outline a standard installation of CDK using the stable release Juju bundles. The standard bundle includes all the components of Kubernetes, but you will have to also follow the [additional configuration][cloud-specific] steps at the end for Kubernetes to function properly - additional components are required to enable Kubernetes to interact with the cloud it is deployed on.
+
+###  Install Juju
+
+If Juju has not already been installed on your system, you need to install it first. For Ubuntu 16.04 and later, and other operating systems which support [snaps][snaps], run the command:
+
+```bash
+sudo snap install juju --classic
+```
+
+For other install options, please see the [Juju documentation][juju-docs].
+
+### Create a controller
+
+Juju requires a *controller*  instance to manage models and deployed applications.
+You can make use of a hosted controller ([JAAS][jaas]) or create one in a cloud of
+your choice. For public clouds, you will need to have added a credential first:
+
+```bash
+juju add-credential
+```
+
+...will step through adding your credential for a specific cloud.
+
+
+The Juju documentation has more information on [adding credentials][credentials] and [configuring a controller][controller-config]
+
+<div class="p-notification--caution">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+To manually install CDK locally using LXD, it is necessary to use a specific container profile before deploying.  See the <a href="/kubernetes/clouds-lxd">notes on LXD</a> before deployment.
```

Comment:
The lxd link doesn't work in the preview.

Reply:
I've commented this warning out for now as the lxd page doesn't exist yet. was going to point to the conjure-up one, but it makes more sense to just do it properly as part of these docs